apps/studio/src/lib/db/clients/BasicDatabaseClient.ts

```diff
@@ -1,9 +1,13 @@
-import { SupportedFeatures, FilterOptions, TableOrView, Routine, TableColumn, SchemaFilterOptions, DatabaseFilterOptions, TableChanges, OrderBy, TableFilter, TableResult, StreamResults, CancelableQuery, ExtendedTableColumn, PrimaryKeyColumn, TableProperties, TableIndex, TableTrigger, TableInsert, NgQueryResult, TablePartition } from '../models';
+import { SupportedFeatures, FilterOptions, TableOrView, Routine, TableColumn, SchemaFilterOptions, DatabaseFilterOptions, TableChanges, OrderBy, TableFilter, TableResult, StreamResults, CancelableQuery, ExtendedTableColumn, PrimaryKeyColumn, TableProperties, TableIndex, TableTrigger, TableInsert, NgQueryResult, TablePartition, TableUpdateResult } from '../models';
 import { AlterPartitionsSpec, AlterTableSpec, IndexAlterations, RelationAlterations, TableKey } from '@shared/lib/dialects/models';
 import { buildInsertQuery } from './utils';
 import { Knex } from 'knex';
-import { DatabaseClient, DatabaseElement } from '../client';
 import { ChangeBuilderBase } from '@shared/lib/sql/change_builder/ChangeBuilderBase';
+import { ConnectionType, DatabaseElement, IDbConnectionDatabase, IDbConnectionServer } from '../types';
+import createLogger from '@/lib/logger';
+import connectTunnel from '../tunnel';
+
+const logger = createLogger('db');
```

Comment:
Oh, we shouldn't be using this. Let's remove it. We should use `electron-log` instead

Reply:
@not-night-but This is the one thing to change before merging, be sure to use `electron-log` and `rawLog.scope('db')`, then kill this old file.